Sources/Sentry/SentryClient.m

```diff
@@ -654,6 +654,12 @@ - (void)setSdk:(SentryEvent *)event
         if (self.options.stitchAsyncCode) {
             [integrations addObject:@"StitchAsyncCode"];
         }
+
+#if SENTRY_HAS_UIKIT
```

Comment:
This might be an iOS only feature. Do you know if it also exists on tvOS?

Reply:
I'm unsure. As this is an experimental feature, I would like to see if we get any data from tvOS projects to find out.